Header header
float32 omega_cmd
float32 posX
float32 posY
bool priority_approach # to raise the priority when far enough away to warrant perpindicular approach
